File: modules/modHelp.py
# modHelp.py
from modules.mod import mod

class module(mod):
	def __init__(self):
		super().__init__()
		self.description = "Show help"
		self.how = "`?help`"

	async def run(self, *args):
		del args
		return "Helpful message!"

File: bot/modules/modHelp.go
package modules

var ModHelp ModuleHelp = ModuleHelp{
	Module{
		Keys: []string{
			"h",
			"help",
		},

		Name:        "Help",
		Description: "",
		How:         "`?help` or `?h`",
	},
}

type ModuleHelp struct {
	Module
}

func (mod *ModuleHelp) Run(args []string) (resp string) {
	return "A helpful message!"
}

func (mod *ModuleHelp) Super() Module {
	return mod.Module
}
